PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Support Files/AppDelegate.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AppDelegate.swift
+//  ControlsLockScreenSampleApp
+//
+//  Copyright © 2017 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+  var window: UIWindow?
+
+
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+    // Override point for customization after application launch.
+    
+    // The audio session can be added in the video view controller.
+    // The session has to be set before to play content.
+    let session: AVAudioSession = AVAudioSession.sharedInstance()
```

Comment:
no need to define type on the left side.

Reply:
Done